examples/volume_plane_rendering.py

```diff
@@ -0,0 +1,230 @@
+"""
+Display one 3-D volume layer using the add_volume API and display it as a plane
+with a widget for modifying plane parameters
+"""
+import numpy as np
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QWidget, QGroupBox, QVBoxLayout, QHBoxLayout, QPushButton
+from scipy.spatial.transform import Rotation as R
+from skimage import data
+from superqt import QLabeledDoubleSlider
+
+import napari
+from napari._qt.qt_event_loop import get_app
+
+blobs = data.binary_blobs(length=64, volume_fraction=0.1, n_dim=3).astype(
+    float
+)
+viewer = napari.Viewer(ndisplay=3)
+viewer.axes.visible = True
+
+# add a volume
+volume_layer = viewer.add_image(
+    blobs, rendering='mip', name='volume', blending='additive', opacity=0.5
+)
+
+# add the same volume and render as plane
+# plane should be in 'additive' blending mode or depth looks all wrong
+plane_layer = viewer.add_image(
+    blobs, rendering='average', name='plane', blending='additive', opacity=0.5
+)
+plane_layer.render_as_plane = True
+
+plane_layer.plane.position = (32, 32, 32)
+plane_layer.plane.normal_vector = (1, 0, 0)
+plane_layer.plane.thickness = 5
+
+# add a point to display plane point
+plane_point_layer = viewer.add_points(np.array([32, 32, 32]), face_color='cornflowerblue',
+                                      name='plane point')
+
+# add a vector to display the plane normal
+normal_data = np.array(
+    [[32, 32, 32],
+     [1, 0, 0]]
+).reshape((-1, 2, 3))
+plane_normal_layer = viewer.add_vectors(normal_data, edge_color='cornflowerblue', length=15,
+                                        edge_width=3, name='plane normal')
+
+# add a point at each corner of the volume to help orient the user
+bounding_box = np.array(
+    [
+        [0, 0, 0],
+        [1, 0, 0],
+        [0, 1, 0],
+        [1, 1, 0],
+        [0, 0, 1],
+        [1, 0, 1],
+        [0, 1, 1],
+        [1, 1, 1],
+    ]
+) * 64
+bounding_box_layer = viewer.add_points(bounding_box, face_color='seagreen', name='bounding_box')
+
+
+class PlaneWidget(QWidget):
+    def __init__(self):
+        super().__init__()
+
+        master_layout = QVBoxLayout(self)
+
+        self.position_sliders = QGroupBox('plane position sliders')
+        position_layout = QHBoxLayout(self.position_sliders)
+
+        self.position_0 = QLabeledDoubleSlider(self)
+        self.position_1 = QLabeledDoubleSlider(self)
+        self.position_2 = QLabeledDoubleSlider(self)
```

Comment:
I'm not seeing sliders here 
![image](https://user-images.githubusercontent.com/6531703/128611448-13944746-6c6d-4421-8201-df7000f074e5.png)